Dockerfile

```diff
@@ -0,0 +1,31 @@
+# ------------------------------------------------------------------------
```

Comment:
Why do we need this Dockerfile at the root ?

Reply:
Bought the file to the docker-auth folder